hil/ext/obm/migrations/mock/df8d9f423f2b_rename_mockobm_table_for_flask.py

```diff
@@ -8,19 +8,17 @@
 
 """
 from alembic import op
-import sqlalchemy as sa
 from hil.model import db
-from hil.flaskapp import app
 
 # revision identifiers, used by Alembic.
 revision = 'df8d9f423f2b'
 down_revision = None
-branch_labels = ('hil.ext.obm.mock',)
+branch_labels = None
```

Comment:
Only the newest migration on a branch should have that branch's label. Otherwise, it should say `None`.